docs/pages/getting-started/3-configure-your-node.rst

```diff
@@ -11,65 +11,66 @@ Prerequisites
 
+To generate the certificate for your own node you need the ``https://github.com/nuts-foundation/nuts-development-network-ca`` repository. It contains handy scripts and the needed key material.
 
-.. note::
+Your node only accepts requests from other nodes which use a certificate issued by one of the trusted CAs. Which CAs to trust is configured by providing a truststore file. The truststore is a PEM file which contains a set of one or more certificates from CAs which the network participants all decided on to trust.
+To learn more about how a Nuts network uses certificates, see the specification `RFC008 <https://nuts-foundation.gitbook.io/drafts/rfc/rfc008-certificate-structure>`_.
 
-    You can start the node without configuring the network, but it won't connect and thus exchange data with other nodes.
 
-YAML Configuration File
-=======================
+To generate certificates for the `development` network perform the following steps:
 
-If you're using a YAML file to configure your node, the following snippet shows an example for the network related configuration:
+.. code-block:: shell
 
-.. code-block:: yaml
+  git clone https://github.com/nuts-foundation/nuts-development-network-ca
+  cd nuts-development-network-ca
+  ./issue-cert.sh development nuts.yourdomain.example
 
-  network:
-    truststorefile: /path/to/truststore.pem
-    certfile: /path/to/certificate-and-key.pem
-    certkeyfile: /path/to/certificate-and-key.pem
-    bootstrapnodes:
-      - example.com:5555
+This results in 3 files:
 
-Node TLS Certificate
-====================
+* ``nuts.yourdomain.example-development.key`` The private key for the node.
+* ``nuts.yourdomain.example-development.pem`` The certificate for the node.
+* ``truststore-development.pem`` The truststore for this (development) network.
 
-To connect to an existing Nuts network you need a TLS certificate which authenticates your node. For the development network
-you can use the ``nuts-network-development-ca`` to directly issue a certificate for your node. The commands below clone
-the required Git repository, generate a private key and issues a certificate, and combines them into a single file:
 
-.. code-block:: shell
+Bootstrap nodes
+===============
 
-  git clone https://github.com/nuts-foundation/nuts-development-network-ca
-  cd nuts-development-network-ca && ./issue-cert.sh localhost
-  cat localhost.key localhost.pem > certificate-and-key.pem
+A bootstrap node is just a normal Nuts node which is available for other nodes to connect to. When you want to join a network, you must approach another network participant and ask for its public endpoint. After connecting, this node will sent an address list of other connected nodes (this requires a Network Identity and NutsComm endpoint, see further down for more information). Your node must connect to the GRPC endpoint which is usually configured on the ``5555`` port.
```

Comment:
A bit of ambiguity: do you need a NutsComm endpoint to receive the addresses or do they need one?